app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
```

Comment:
You can make something like cmd, old_file, new_file = command to improve code readability